classes/import-export.php

```diff
@@ -514,6 +515,24 @@ public function import_csv() {
 		}
 
 		/**
+		 * Gets import path.
+		 *
+		 * @param $file
+		 *
+		 * @return string
+		 */
+		function get_import_path( $file ) {
+			$upload_dir = wp_upload_dir();
+
+			// fallback to original path...?
+			if ( isset( $upload_dir['error'] ) && ! empty( $upload_dir['error'] ) ) {
+				return WP_PLUGIN_DIR . '/sm-temp-csv-' . $file;
+			}
+
+			return $upload_dir['path'] . '/sm-temp-csv-' . $file;
+		}
+
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->